61-rotate-linked-list.py

```diff
@@ -0,0 +1,48 @@
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.next = None
+
+class Solution:
+  def rotateRight(self, head: ListNode, k: int) -> ListNode:
+    '''
+    Time complexity: O(2n)
+    Space complexity: O(1) ??
```

Comment:
Time is just `n` (`n` shows that it's just "linear" even if we may check the list 10 times) and space is `1` indeed. I believe we use the same amount of memory no matter how long the list is, don't we?

Reply:
Yes, both is true indeed. Updated.